src/OmniSharp.Roslyn.CSharp/Services/DocumentationConverter.cs

```diff
@@ -131,10 +132,40 @@ private static string GetCref(string cref)
 
         public static DocumentationComment GetStructuredDocumentation(string xmlDocumentation, string lineEnding)
         {
-            if (string.IsNullOrEmpty(xmlDocumentation))
-                return null;
             return DocumentationComment.From(xmlDocumentation, lineEnding);
         }
+
+        public static DocumentationComment GetStructuredDocumentation(ISymbol symbol, string lineEnding = "\n")
+        {
+            switch (symbol)
+            {
+                case IParameterSymbol parameter: return DocumentationComment.WithSummaryText(DocumentationHelper.GetParameterDocumentation(parameter, lineEnding));
+                case ITypeParameterSymbol typeParam: return DocumentationComment.WithSummaryText(DocumentationHelper.GetTypeParameterDocumentation(typeParam, lineEnding));
+                case IAliasSymbol alias: return DocumentationComment.WithSummaryText(DocumentationHelper.GetAliasDocumentation(alias, lineEnding));
+                default: return GetStructuredDocumentation(symbol.GetDocumentationCommentXml(), lineEnding);
+            }
+        }
+    }
+
+    public static class DocumentationHelper
+    {
+        public static string GetParameterDocumentation(IParameterSymbol parameter, string lineEnding = "\n")
```

Comment:
What's the advantage of putting these in another class versus inside DocumentationConverter? Do you imagine other code using these methods?

Reply:
NVM, used below. Can you move this class to its own file?